### ISSUE TITLE
Jinja2 terms on basic/intro template

### ISSUE THREAD

**Issue by vfulco** [tarbell-project/tarbell#379]
Sending this sheepishly as I am blown away by how extraordinary the body of work is in total.  I've been able to read thru the tutorial a few times and installed and created a newproject without any errors surfacing.  Change the 'headline' in the linked google spreadsheet, local dev server is running and I can access the page, using Atom IDE open the index.html file within the newly created folder, can't find any reference to a <headline> tag.  There are jinja2 terms extending _base.html and segregating block content but no tags for key/values otherwise.  What am I missing?

{% extends "_base.html" %}

{% block content %}
<div class="container">
  <div class="jumbotron">
    <h1>Welcome to your new project!</h1>
    <p>Replace this message with your own content in <code>{{ PROJECT_PATH }}/index.html</code>.</p>
    <p>Learn more on the <a href="_example.html">example page</a>.</p>
  </div>
</div>
{% endblock content %}

Separately working from Asia and been having cloudflare issues with other services but I know that shouldn't be an issue. 

Thanks in advance.  The work is truly phenomenal.

**Comment by vfulco** [comment]
Ahh, github is styling the html code but you see what I mean, no jinja2 tags other than the project-path one.

**Comment by eads** [comment]
good call... we should fix that to show more about the key/value stuff.

for the moment, do you understand how to get key -> value pairs into the
spreadsheet and then display them in the template?

On Thu, Oct 8, 2015 at 5:08 AM, Vincent C Fulco <notifications@github.com>
wrote:

> Ahh, github is styling the html code but you see what I mean, no jinja2
> tags other than the project-path one.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tarbell-project/tarbell/issues/379#issuecomment-146465261>
> .
>



-- 
David Eads | http://recoveredfactory.net

"Medical statistics will be our standard of measurement: we will weigh life
for life and see where the dead lie thicker, among the workers or among the
privileged." -- Rudolf Virchow

**Comment by vfulco** [comment]
I do and will proceed that way.  Wanted to make sure the I wasn't missing anything as a relative noob to Flask and this more sophisticated, complete framework.  Hope to contribute, however modestly once I get up the curve a bit.  Thanks for the timely response and again the excellent work.
